ember-cli/2017-10/october-10.md

```diff
@@ -0,0 +1,8 @@
+# ember-cli Meeting Notes - 2017-10-10
+
+## Attendees
```

Comment:
Please reply here with agenda items

Reply:
SIGUSR1 or equiv for triggering rebuild cc @rwjblue

Reply:
follow-up strategies rfc

Reply:
release post-mortem

Reply:
ember-cli-update in core status